app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -76,6 +76,36 @@ def bulk_search
         render :json => ret
       end
 
+      api :PUT, '/tasks/bulk_resume', 'Resume all paused error tasks'
+      def bulk_resume
+        scope = resource_scope_for_index.select('DISTINCT foreman_tasks_tasks.*')
+        scope = scope.where(:state => :paused)
+        scope = scope.where(:result => :error)
```

Comment:
I'll add some CLI options for the resume command. Any suggestions? I was thinking of --search (resume all that match the search) and --no-op (just print out results found, don't actually resume).

I do think the default should probably just be the ones in paused/error